vipps-checkout-api-quick-start.md

```diff
@@ -19,59 +27,161 @@ Use the Checkout API to create a checkout session and retrieve session informati
 
-Review
-[Quick start guides](https://developer.vippsmobilepay.com/docs/quick-start-guides) for information about getting your test environment set up.
+**Important:** The examples use standard example values that you must change to
+use *your* values. This includes API keys, HTTP headers, reference, etc.
 
-### Step 1: Get the collection and environment
+## Your first Payment
 
-Save the following files to your computer:
+### Step 1 - Setup
 
-* [Checkout API Postman collection](/tools/vipps-checkout-api-postman-collection.json)
-* [Global Postman environment](https://raw.githubusercontent.com/vippsas/vipps-developers/master/tools/vipps-api-global-postman-environment.json)
+<Tabs
+defaultValue="curl"
+groupId="sdk-choice"
+values={[
+{label: 'curl', value: 'curl'},
+{label: 'Postman', value: 'postman'},
+]}>
+<TabItem value="postman">
 
-### Step 2: Import the Postman files
+**Please note:** Postman is discontinuing their offline version. Use only your test keys and delete them after testing.
+Ensure that your company allows for cloud use before continuing.
 
-1. In Postman, click *Import* in the upper-left corner.
-1. In the dialog that opens, with *File* selected, click *Upload Files*.
-1. Select the two files you have just downloaded and click *Import*.
+If you wish to use Postman, import the following files:
 
-### Step 3: Set up Postman environment
+* [Checkout API Postman collection](/tools/vipps-checkout-api-postman-collection.json)
+* [Global Postman environment](https://github.com/vippsas/vipps-developers/blob/master/tools/vipps-api-global-postman-environment.json)
+
+In Postman, tweak the environment with your own values (see
+[API keys](https://developer.vippsmobilepay.com/docs/common-topics/api-keys/)):
 
-1. Click the down arrow, next to the "eye" icon in the top-right corner, and select the environment you have imported.
-1. Click the "eye" icon and, in the dropdown window, click `Edit` in the top-right corner.
-1. Fill in the `Current Value` for the following fields to get started. For the first keys, go to *Vipps Portal* > *Utvikler* > *Test Keys*.
-   * `client_id` - Merchant key is required for getting the access token.
-   * `client_secret` - Merchant key is required for getting the access token.
-   * `Ocp-Apim-Subscription-Key` - Merchant subscription key.
-   * `merchantSerialNumber` - Merchant ID.
-   * `internationalMobileNumber` - The mobile number for the test app profile you have received or registered, including country code. Do not include the `+`.
+* `client_id` - Merchant key required for getting the access token.
+* `client_secret` - Merchant key required for getting the access token.
+* `Ocp-Apim-Subscription-Key` - Merchant subscription key.
+* `merchantSerialNumber` - Merchant ID.
+* `internationalMobileNumber` - The MSISDN for the test app profile you have received or registered. This is your test mobile number *including* country code.
```

Comment:
Does not apply to our api

Reply:
Fixed.

Reply:
Should be removed

Reply:
"internationalMobileNumber" right? It should be gone in the new version now.